reference/promotions.v3.yml

```diff
@@ -349,6 +351,42 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+
+  '/promotions/codes':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    get:
+      tags:
+        - Coupon Code (Single)
```

Comment:
The code query param is optional here right? So this is actually a listing endpoint?